code/game/objects/machinery/vending/new_marine_vendors.dm

```diff
@@ -422,7 +422,7 @@ GLOBAL_LIST_INIT(marine_selector_cats, list(
 		/obj/item/armor_module/storage/general = list(CAT_MOD, "General Purpose Storage Module", 0, "black"),
 		/obj/item/armor_module/storage/engineering = list(CAT_MOD, "Engineering Storage Module", 0, "black"),
 		/obj/item/storage/pouch/shotgun = list(CAT_POU, "Shotgun shell pouch", 0, "black"),
-		/obj/item/storage/pouch/construction = list(CAT_POU, "Construction pouch", 0, "orange"),
+		/obj/item/storage/pouch/construction/full = list(CAT_POU, "Construction pouch", 0, "orange"),
```

Comment:
this plus the the price of mats in vendor is too much

Reply:
so back to empty construction pouch?

Reply:
yes IMO

Reply:
ok i put it back to empty variant